### PR TITLE
Added unique=True in category name to resolve Issue #2472

### PR DESCRIPTION
**Pull request by lostwarrior404**
fixing issue #2472.

### CODE REVIEW

File: openslides/motions/models.py [hunk 1]
```diff
@@ -703,7 +703,7 @@ class Category(RESTModelMixin, models.Model):
     """
     access_permissions = CategoryAccessPermissions()
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=255, unique=True)
```
Comment (FinnStutzenstein) [inline openslides/motions/models.py]: @lostwarrior404 I think for this change in the databasestructure a migration is necessary. What happens if there are already two categories with the same name in the db?

**Comment by emanuelschuetze** [comment]
@lostwarrior404 ping (Let us know if you need help)

**Comment by lostwarrior404** [comment]
Sorry for the late reply I'll try to resolve this by tonight.

**Comment by lostwarrior404** [comment]
When I try to apply migrations using command
`./manage.py makemigrations openslides.motions --setting=openslides.global_settings`
I get an error `ImproperlyConfigured("The SECRET_KEY setting must not be empty.")`.
Am i doing something wrong? Can someone explain how to proceed towards solving this?

**Comment by normanjaeckel** [comment]
In a development version you can just run `python manage.py createsettings` to get a development settings with random secret key and the like. See the new directory personal_data. The you can run `python manage.py makemigrations` to get a new migrations file.

**Comment by lostwarrior404** [comment]
@normanjaeckel Can you review the requested changes again?

**Comment by emanuelschuetze** [comment]
@lostwarrior404 Ping. When can you  finish this pull request? Please let us know if you need help.

**Comment by emanuelschuetze** [comment]
@lostwarrior404 Do you need any help here to complete this? Otherwise we have to close this (unfinished and outdated) pull request.

**Comment by normanjaeckel** [comment]
Implementation is not complete. We close this pull request. Feel free to reopen with a new proposal (rebased on top of current master) or even open a new pull request. Thanks in advance.

**Pull request closed without merging**